packages/sources/svelte/rollup.config.js

```diff
@@ -22,6 +22,10 @@ const src = {
       folder: 'overlays',
       components: ['VtmnPopover'],
     },
+    {
+      folder: 'navigation',
+      name: 'VtmnSearch',
+    },
```

Comment:
Can you please use alphabetical order for folders and object with an array in `components` field instead of `name`? Thanks!